tests/lib/rules/require-default-props.js

```diff
@@ -1718,6 +1728,17 @@ ruleTester.run('require-default-props', rule, {
           column: 3
         }
       ]
+    },
+    {
+      code: [
+        'import type { ImportedType } from "fake";',
+        'type Props = ImportedType;',
+        'function Hello(props: Props) {',
+        '  return <div>Hello {props.name.firstname}</div>;',
+        '}'
+      ].join('\n'),
+      parser: 'babel-eslint',
+      errors: []
```

Comment:
We should update this to return errors so we can test for it, perhaps in the event that the type isn't defined. Maybe something like this:

```
function Hello (props: UndefinedVariable) {
    return <div>Hello {props.name.firstname}</div>;
}
```